draft-packages/select/package.json

```diff
@@ -34,7 +34,7 @@
   "dependencies": {
     "@kaizen/component-library": "^12.4.2",
     "@kaizen/deprecated-component-library-helpers": "^2.5.0",
-    "@kaizen/draft-button": "^5.6.2",
+    "@kaizen/button": "^1.0.1",
```

Comment:
This one is interesting. Seems like it's only used in the `.elm` and not in the main component 🤔
```elm
-- L43
import KaizenDraft.Button.Button as Button
```

The options I see for this would be:
1. Remove the dependency if the elm file does not use it
2. Update the elm file to point to the new Button
3. Revert this change in `package.json` and leave a comment that it's only used for elm

Reply:
Based off the catch up today I'll add both in this pattern:

```
    "@kaizen/button": "^1.0.1",
    "//": "The draft dependency should remain here until this is moved out of draft packages ",
    "@kaizen/draft-button": "^5.6.2",
```

Reply:
I don't believe this will be treated nicely (apparently things explode; I guess it will try to install a `//` package with this string version).

Here's the pattern from root's `package.json` :)

```
  "dependencies": {
    "node-sass": "6.0.1",
  },
  "dependenciesComments": {
    "node-sass": "version locked to keep consistency with gatsby site",
  },
```